src/WebJobs.Script/Description/Script/ScriptFunctionInvoker.cs

```diff
@@ -104,7 +100,7 @@ public static bool IsSupportedScriptType(ScriptType scriptType)
                 }
             }
 
-            TraceWriter.Info(string.Format("Function started (Id={0})", invocationId));
+            TraceWriter.Info(string.Format("Script Function started (Id={0})", invocationId));
```

Comment:
Please leave these as just "Function started", including in your PowerShell invoker. The user knows the type of their script :)